resolwe_bio/processes/reads_processing/alignmentsieve.py

```diff
@@ -0,0 +1,151 @@
+"""Filter BAM files using alignmentSieve from deeptools"""
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Process,
+    SchedulingClass,
+    Persistence,
+    Cmd,
+    StringField,
+    IntegerField,
+    FileField,
+    DataField,
+)
+
+
+class AlignmentSieve(Process):
+    """Filter alignments of BAM files according to specified parameters
+
+    Optionally, it can output a BEDPE file. Program is bundled with
+    deeptools. See
+    https://deeptools.readthedocs.io/en/develop/content/tools/alignmentSieve.html
```

Comment:
Something like `[documentation](https://deeptools.readthedocs.io/en/develop/content/tools/alignmentSieve.html) ` could be used here.